packages/@aws-cdk/aws-iotevents/lib/input.ts

```diff
@@ -8,11 +9,55 @@ import { CfnInput } from './iotevents.generated';
 export interface IInput extends IResource {
   /**
    * The name of the input
+   *
    * @attribute
    */
   readonly inputName: string;
+
+  /**
+   * The ARN of the input
+   *
+   * @attribute
+   */
+  readonly inputArn: string;
+
+  /**
+   * Grant the putting message permission to the given IAM principal (Role/Group/User).
+   *
+   * @param grantee the principal
+   */
+  grantPutMessage(grantee: iam.IGrantable): iam.Grant
```

Comment:
What do you actually think of renaming this to `grantWrite()`? Maybe that's a better name, and will allow us to expand the set of granted permissions here as needed in the future?

Reply:
Sound's good!
> will allow us to expand the set of granted permissions here as needed in the future?

That's reasonable!

Reply:
So... why is this `grantPushMessage()` still? 😛

Reply:
Oops..